devel/boost/Portfile

```diff
@@ -255,36 +238,78 @@ http://www.boost.org/doc/libs/${distver}/libs/regex/doc/html/boost_regex/ref/sub
     }
 }
 
-if {[mpi_variant_isset]} {
-
-    # There is a conflict with debug support.
-    # The issue has been reported to both the MacPorts team and the boost team, as per:
-    # <http://trac.macports.org/ticket/23667> and <https://svn.boost.org/trac/boost/ticket/4461>
-    if {[variant_isset debug]} {
-        return -code error "+debug variant conflicts with mpi"
+subport boost-numpy {
+    description Boost.Numpy library
+    long_description ${description}
```

Comment:
You also need
>     depends_lib port:boost
here to get the `require_active_variants` to work. You can choose to `append`, but I think just setting is cleaner. Not sure which is really correct though.

Reply:
Fixed. I checked that boost is indeed the only runtime dependency:
```
> otool -L /opt/local/lib/libboost_numpy3-mt.dylib 
/opt/local/lib/libboost_numpy3-mt.dylib:
	/opt/local/lib/libboost_numpy3-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
	/opt/local/lib/libboost_python3-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
	/opt/local/lib/openmpi-mp/libmpi_cxx.40.dylib (compatibility version 41.0.0, current version 41.0.0)
	/opt/local/lib/openmpi-mp/libmpi.40.dylib (compatibility version 41.0.0, current version 41.0.0)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 400.9.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1252.0.0)
```

Reply:
Hmm ... I wonder if the MPI variant also needs to be the same. I don't use it, but I see the linkage in your `otool` output & that makes me wonder. The output of mine is same as yours just without the MPI libraries, so I think you're right that we're safe using just boost as a dependency.